src/OpenTelemetry.Exporter.Prometheus.AspNetCore/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+* Added option to disable _total suffix addition to counter metrics in
```

Comment:
No need to mention "in Prometheus exporter". This CHANGELOG file is specific to Prometheus exporter.

Reply:
fixed